c/tskit/core.c

```diff
@@ -287,6 +287,18 @@ tsk_strerror_internal(int err)
         case TSK_ERR_UNSORTED_MUTATIONS:
             ret = "Mutations must be provided in non-decreasing site order";
             break;
+        case TSK_ERR_MUTATION_TIME_YOUNGER_THAN_NODE:
+            ret = "Mutations must have a time equal to or greater than that of their "
```

Comment:
This doesn't quite parse (it's implying the time must be greater than TSK_MUTATION_UNKNOWN_TIME). How about "A mutation's time must be >= the node time, or be marked as 'unknown'".

This error message will be shown in Python and C, so we don't want to use the C macro.

The other messages aren't quite right either.

Reply:
Fixed https://github.com/tskit-dev/tskit/pull/672/commits/6180ad53b8098de42fa9b6c01ef77a0448d36d26